### ISSUE TITLE
Hitting PrintScreen [PrtSc] should by default "Copy Screen To Clipboard".  

### ISSUE THREAD

**Issue by MiY4Gi** [linuxmint/Cinnamon#1622]
I just learnt now that the PrtSc button automatically sends a screenshot to the /home/pictures folder, however it does not copy the image to clipboard. This is not intuitive, especially considering that users have long gotten used to treating PrtSc as "copy", and then "pasting" the data into an image editing programme like MS Paint or Gimp. I learnt that Ctrl+PrtSc copies to clipboard. Although I like the convenience of having my Screenshot automatically saved to file, considering that it's a lot faster than the old method, not being told about this change makes it very INconvenient.

My suggestion for Cinnamon devs is for them to do one of the following: 
1. Change the default action for [PrtSc] to "Copy Screen to Clipboard" and set [Ctrl+PrtSc] to "Save to /Home/Pictures". Also, displaying a notification about the [Ctrl+PrtSc] shortcut when [PrtSc] is pressed would teach users about this (new?) feature.
2. Change the default action for [PrtSc] to both "Copy Screen to Clipboard" AND "Save to /Home/Pictures".
3. Change the default action for [PrtSc] to display a prompt asking user what to do with image, and then prompt the user to choose his preferred "default action".

**Comment by dandv** [comment]
One of the first things that I change after I install Mint, is to make PrintScreen place the screenshot in the clipboard, Alt+PrintScreen copy to the clipboard (oh well, [when it works](https://github.com/linuxmint/cinnamon/issues/7124)), and Shift+PrintScreen to copy to the clipboard (which [doesn't work either](https://github.com/linuxmint/cinnamon/issues/9824)). This is what I've gotten used to after ~25 years of desktop computing, the last 5 of which on Ubuntu.

Savings screenshots automatically is something I don't like or use, because,
1. The vast majority of the time, I'm pasting the screenshot somewhere else - Slack, GitHub, Discord etc. or a graphics editing program.
2. I really *never* need the full screenshot. It has unwanted and sensitive information, so I always edit it. I fire up Pinta/Gimp/whatever, and paste it. Simple as that. Don't have to hunt for a file named "Screenshot12345" out of a sea of files named "ScreenshotXXXXX"
3. The name is autogenerated. If I want a screenshot saved, I'll save it from Gimp/whatever, with a meaningful name.

So TL;DR - another vote for the shorter keyboard shortcuts copying to the clipboard, *not* saving.
